charts/ingress-nginx/values.yaml

```diff
@@ -16,16 +16,21 @@ commonLabels: {}
 controller:
   name: controller
   image:
+    ## This directory is for tools and things that are used to administer the GCR repositories used to publish official container images for Kubernetes.
```

Comment:
Not sure what you mean by directory here.

Reply:
I will replace that word with something that would make sense. is everything going correct besides that ?